### PR TITLE
Replaced Classes with functional components

### PR DESCRIPTION
**Pull request by tushhr**
Issue: #5

How can I update `forms`, `loginForm`? loginForm is extending forms, so how can I implement that using Functions...Can I?

Also, look carefully for the `user` stuff. I'm not exactly sure about that...

### CODE REVIEW

**Comment by tushhr** [comment]
@MananJethwani!

**Comment by MananJethwani** [comment]
@tushhr leave what can be converted to functional components for now....the form class will help us with inheritance if needed.

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -11,76 +11,164 @@ import jwtDecode from "jwt-decode";
 import "react-toastify/dist/ReactToastify.css";
 import "./App.css";
 
-class App extends Component {
-  state = {
-    isModalOpen: false,
-    isModalOpen2: false,
-  };
+// class App extends Component {
```
Comment (MananJethwani) [inline frontend/src/App.jsx]: please remove comment line the code should always be clean as it is going to be merged in master

File: frontend/src/commponents/semester.jsx [hunk 2]
```diff
@@ -1,25 +1,47 @@
...
+  const renderSuffix = () => {
+    if (props.year === 1) {
+      return "st"
+    } else if (props.year === 2) {
+      return "nd"
+    } else if (props.year === 3) {
+      return "rd"
     } else {
-      return "th";
+      return "th"
```
Comment (MananJethwani) [inline frontend/src/commponents/semester.jsx]: don't remove semicolon it helps clarify the code also put proper semicolons where ever possible.

**Review by MananJethwani — CHANGES_REQUESTED** [review]
@tushhr thanks for the PR and sorry for the late response won't happen again please change as requested here

File: frontend/src/App.jsx (hunk 1 above)
Comment (tushhr) [reply frontend/src/App.jsx]: Yeah definitely, I just put them in the comment for the ease of the review. Anyway, I will remove them.

**Comment by tushhr** [comment]
> Sorry for the late response won't happen again

It's fine. I have done the changes as requested.

**Review by MananJethwani — APPROVED** [review]
thanks @tushhr

**Pull request merged**